### PR TITLE
Feature/kas 3436 refactor van stateless helper methods in agenda service om backend service aan te roepen naar utils

### PR DESCRIPTION
**Pull request by sergiofenoll**
Moves all the agenda-approve related methods from `agenda-service` to a new util `agenda-approval`. These are only used in `AgendaVersionActions`, i.e. the first (blue) dropdown in the agenda overview which deals with approving, closing, deleting, ... agendas and meetings.

The rest of the methods in the agenda service do seem to be stateful, so they're untouched.

### CODE REVIEW

**Review by erikap — APPROVED** [review]

File: app/components/agenda/agenda-header/agenda-version-actions.js [hunk 1]
```diff
@@ -380,18 +386,18 @@ export default class AgendaAgendaHeaderAgendaVersionActions extends Component {
...
     }
     try {
-      const lastapprovedAgenda = await this.agendaService.deleteAgenda(
-        this.args.currentAgenda
-      );
-      if (lastapprovedAgenda) {
+      const lastApprovedAgendaId = await deleteAgenda(this.args.currentAgenda);
+      let lastApprovedAgenda;
+      if (lastApprovedAgendaId) {
+        lastApprovedAgenda = await this.store.findRecord('agenda', lastApprovedAgendaId);
```
Comment (ValenberghsSven) [inline app/components/agenda/agenda-header/agenda-version-actions.js]: `lastApprovedAgenda` does not seem used outside of the following `if` statement so why not a `const` inside the `if`?

**Review by ValenberghsSven — APPROVED** [review]
Changed a `let` with only 1 assignment to a `const`
Approved if jenkins run is green

File: app/components/agenda/agenda-header/agenda-version-actions.js (hunk 1 above)
Comment (ValenberghsSven) [reply app/components/agenda/agenda-header/agenda-version-actions.js]: Changed it myself to shorten the review process

**Comment by ValenberghsSven** [comment]
ga nog even updaten met dev

**Comment by ValenberghsSven** [comment]
flaky test flaky, rest ok. Merging

**Pull request merged**